harmonisation/specimen/taxonomicTerms.md

```diff
@@ -0,0 +1,62 @@
+# Taxonomic terms
+
+This is a set of terms which are combined in this single file.
+We combined these terms as we use a similar approach to harmonised them.
+Additionally, it would unnecessarily grow the amount of terms files in this repository.
+
+## openDS
+These terms will be harmonised for taxonomy.
+- `dwc:class`
+- `dwc:family`
+- `dwc:genus`
+- `dwc:infraspecificEpithet`
+- `dwc:kingdom`
+- `dwc:order`
+- `dwc:phylum`
+- `dwc:scientificNameAuthorship`
+- `dwc:specificEpithet`
+- `dwc:taxonRank`
+- `dwc:typeStatus`
+
+## Mapping
+Before searching for the taxonomy term in the below, terms will first check if a default value or explicit mapping has been set.
```

Comment:
'in the below' of what? terms? should then be placed also before the comma right?

Reply:
:+1: